cabot/metricsapp/models/base.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from cabot.cabotapp.models import CHECK_TYPES, Service, StatusCheck
+from cabot.metricsapp.api import run_metrics_check
+
+
+class MetricsSourceBase(models.Model):
```

Comment:
I haven't merged them yet, changes from only this are here: https://github.com/Affirm/cabot/pull/35/commits/5c0fc52c8ba7a8bf400c122c01ee889d51396b72

I can merge the base models today though